gtfs-netex-test/aux_always_add_id_version.py

```diff
@@ -40,8 +41,12 @@ def add_id_and_version(input_file, output_file):
                 if 'version' not in element.attrib:
                     element.set('version', 'any')
 
+
+        # Remove the namespace prefix for the specific namespace
+        ET.register_namespace('', 'http://www.netex.org.uk/netex')
         # Write the modified XML back to file
-        tree.write(output_file,encoding='UTF-8', xml_declaration=True, default_namespace=None)
+
+        tree.write(output_file, encoding='UTF-8', xml_declaration=True, default_namespace="")
```

Comment:
```suggestion
        tree.write(output_file, encoding='UTF-8', xml_declaration=True, default_namespace="")
```

Reply:
Are you sure about default_namespace="" ?

Reply:
It worked. But perhaps I should check.